reya-rust-sdk/src/http_provider.rs

```diff
@@ -392,6 +392,57 @@ impl HttpProvider {
         eyre::Ok(receipt)
     }
 
+    pub async fn trigger_auto_excahnge(
+        &self,
+        signer: PrivateKeySigner,
+        params: data_types::TriggerAutoExchangeParams,
+    ) -> eyre::Result<TransactionReceipt> // return the transaction receipt
```

Comment:
In develop branch this is changed to return the tx_hash, receipts can be retrieved with a separate call.

Reply:
resolved in [098a64f](https://github.com/Reya-Labs/reya-rust-sdks-monorepo/pull/23/commits/098a64f58f4b9760e1ead716c1a7e27e95f83f5f)